scenarios/aws/vm/ec2os/os.go

```diff
@@ -16,8 +16,9 @@ type OS interface {
 type Type int
 
 const (
-	WindowsOS Type = iota
-	UbuntuOS       = iota
+	WindowsOS           Type = iota
+	UbuntuOS                 = iota
+	AmazonLinuxDockerOS      = iota
```

Comment:
You probably also want to support it for [the invoke task](https://github.com/DataDog/test-infra-definitions/blob/1c7e762fb2ad06b6da8f61620f28cca9cf1f079e/scenarios/aws/vm/run.go#L65-L84).